semantics/language/translation/decl/tagged.k

```diff
@@ -27,7 +27,8 @@ module C-DECL-TAGGED
           => addTag(X, makeStructFieldInfo(L))
           requires L =/=K .List
           [structural]
-     rule (.K => UNDEF("CDT1", "structs cannot be empty.", "6.7.2.1") )
+    //Undefinedness occurs when there are no named members in a struct. An empty struct is a contraint Violation. 
+     rule (.K => CV("CDT1", "structs cannot be empty.", "6.7.2.1") )
```

Comment:
as far as I can tell this is an undefined behavior, section 6.7.2.1:8, J.2 item 61.